db/seeds.rb

```diff
@@ -2,17 +2,42 @@
 I18n.config.available_locales = %w(fr en)
 I18n.reload!
 Faker::Config.locale = :fr
+employer = Employer.create! name: 'DIRISI', code: 'DRI'
 
-administrator = Administrator.new email: 'pipo@molo.fr',
+bantAdmin = Administrator.new email: 'pipo@molo.fr',
```

Comment:
ruby style is snakecase. Example: bant_admin

Reply:
ok